common_features.mk

```diff
@@ -225,6 +225,12 @@ ifeq ($(strip $(ENCODER_ENABLE)), yes)
     OPT_DEFS += -DENCODER_ENABLE
 endif
 
+ifeq ($(strip $(HAPTIC_ENABLE)), yes)
```

Comment:
if this supports multiple answers, it may be worth changing this to be handled like the `BOOTMAGIC_ENABLE` or `RGB_MATRIX_ENABLE` options (eg, allow for an array of answers)

Reply:
good idea!